brainiak/fcma/preprocessing.py

```diff
@@ -11,144 +11,22 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
-"""Full Correlation Matrix Analysis (FCMA)
-
-FCMA related IO routines
-"""
+"""FCMA preprocessing."""
 
 # Authors: Yida Wang
```

Comment:
We do not have BrainIAK guidelines for listing authors in files. This is purposeful. Other organizations producing open-source software even prohibit listing authors in source files. I am not against people adding their name, but I see little need for that, given the full authorship history Git gives us, so I will not add my name.